database/createDb.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE users (id integer PRIMARY KEY, username STRING NOT NULL);
+CREATE TABLE files (id integer PRIMARY KEY, filename STRING NOT NULL, data BLOB, owner integer, FOREIGN KEY(owner) REFERENCES users(id));
```

Comment:
I'm not sure if that's necessary, since each file already has a unique ID it should be fine as far as the database goes. As far as user experience goes, Google Docs lets you create multiple docs with the same name so ¯\_(ツ)_/¯